sigil.lic

```diff
@@ -0,0 +1,153 @@
+custom_require.call(%w[common common-arcana common-travel drinfomon equipmanager events])
+
+class Sigil
+  include DRC
+  include DRCA
+  include DRCT
+
+  def initialize
+    settings = get_settings
+    @target = variable[1]
+    @target_precision = settings.sigil_min_precision_threshold
+    @sigil_book = settings.sigil_book
+
+    @primary_sigils = ['abolition','congruence','induction','permutation','rarefaction']
+    @secondary_sigils = ['antipode','ascension','clarification','decay','evolution','integration','metamorphosis','nurture','paradox','unity']
+    
+    Flags.add('precision', /Precision:(\d+)/)
+    Flags.add('clarity', /Clarity:(\d+)/)
+    Flags.add('danger', /Danger:[ -]+(\[*]{0,20})/)
+    Flags.add('sanity', /(?=Sanity:([ ]*)(\*{0,20}))/)
+    Flags.add('resolve', /(?=Resolve:([ ]*)(\*{0,20}))/)
+    Flags.add('focus', /(?=Focus:([ ]*)(\*{0,20}))/)
+    Flags.add('moresigils', "Remnants of the sigil pattern linger,")
+
+    @equipment_manager = EquipmentManager.new
+    @equipment_manager.empty_hands
+    
+    @current_precision = 0
+
+    harvest_sigil
+  end
+
+  
+  def find_sigil?
+    case line = bput('perceive sigil', @primary_sigils, @secondary_sigils, 'Roundtime', 'Having recently been searched')
+    when 'Having recently been searched'
+      exit
```

Comment:
(Not blocking merging) If there's a cooldown on searching, that might be nice to track (in either of the scripts) and exit early.

Reply:
There's a cooldown on searching but it's per room and pretty short (5 minutes?).  I shuffle the sigil rooms in sigil-buddy to prevent two people running the same path with the second person never finding anything because the room was too recently searched.